specutils/analysis/template_comparison.py

```diff
@@ -224,29 +223,28 @@ def template_redshift(observed_spectrum, template_spectrum, min_redshift, max_re
     redshifted_spectrum: :class:`~specutils.Spectrum1D`
         A new Spectrum1D object which incorporates the template_spectrum with a spectral_axis
         that has been redshifted using the final_redshift.
+    chi2_list : `list`
```

Comment:
I think this should be an `ndarray` instead of a list. That will make it a lot easier to work with if it's multidimensional.

Relatedly, I'm having a bit of trouble following the logic: is the resulting array is of dimension equal to the spectral template set, or is it a *2D* array that's n_template x n_redshift ? I think the latter is the eventual goal, but if we're not quite there it could be a follow-on.

Reply:
In some out-of-band discussion, @ibusko said he'd write a test for the `SpectrumCollection` case and see what happens.

My preference is that it returns a 2D array as indicated above (or the transpose)